SwissDataScienceCenter/renku-ui
# refactor(state): use redux model to store user data

User data have been migrated to the global model.
Most data are now part of the `user` sub-schema, while projects related data are in the `projects` sub-schema.

This required a small re-factory of multiple components. Please take some time to test it.

***Hot to test it***
A preview is available on https://lorenzotest.dev.renku.ch/
Try to navigate (keeping the javascript console open to notice hidden errors) and do some actions involving user-related content, e.g. :
* star/unstar project
* create new project
* search through personal/starred projects
* starting new interactive environment

fix #632
